src/sage/matroids/linear_matroid.pyx

```diff
@@ -129,7 +129,10 @@ from sage.matrix.matrix2 cimport Matrix
 import sage.matrix.constructor
 from sage.matrix.constructor import matrix
 from copy import copy, deepcopy
-from sage.rings.all import ZZ, QQ, FiniteField, GF
+from sage.rings.integer_ring import ZZ
+from sage.rings.rational_field import QQ
+from sage.rings.finite_rings.finite_field_constructor import FiniteField
+from sage.rings.finite_rings.finite_field_constructor import FiniteField as GF
```

Comment:
These imports seem to be duplicates

Reply:
Yes, and the first one was not even used. I've cleaned it up now